.travis.yml

```diff
@@ -42,6 +42,10 @@ matrix:
       dist: xenial    # Required for Python 3.7
       sudo: required  # travis-ci/travis-ci#9069
       env: BACKEND=cpp
+    - python: 3.7
+      dist: xenial    # Required for Python 3.7
+      sudo: required  # travis-ci/travis-ci#9069
+      env: LIMITED_API=1
```

Comment:
You could also just define `LIMITED_API=--limited-api` here, and then pass it directly as a command line argument below.

Reply:
Done!